Servers/services/reportService.ts

```
@@ -95,6 +96,8 @@ export async function getReportData(
     case ReportType.VENDOR_REPORT:
       markdownFormattedData = await getVendorReportMarkdown(projectId, reportBody)
       break;
+    case ReportType.ANNEXES_REPORT:
+      markdownFormattedData = await getAnnexesMarkdown(frameworkId, reportBody)
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)